internal/server/stat/golden/get_stat_date_within_place/CA_County.json

```diff
@@ -0,0 +1,156 @@
+{
+  "data": {
```

Comment:
@n-h-diaz @chejennifer This is how the response looks like.

Reply:
this looks good to me!

Reply:
got it thanks! will switch to using dates from this!